### PR TITLE
Feature/add editconfiguration endpoint

### PR DESCRIPTION
**Pull request by asyagraf**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/958
add endpoint to controller, command, method to repository

### CODE REVIEW

File: src/AdminService.Business/Commands/EditServiceConfigurationCommand.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    public async Task<OperationResultResponse<bool>> ExecuteAsync(List<Guid> servicesId)
+    {
+      if (!await _accessValidator.IsAdminAsync())
+      {
+        return _responseCreator.CreateFailureResponse<bool>(HttpStatusCode.Forbidden);
+      }
+
+      if (servicesId.Count == 0 || servicesId is null)
```
Comment (MariyaSin) [inline src/AdminService.Business/Commands/EditServiceConfigurationCommand.cs]: use Any()

File: src/AdminService.Business/Commands/EditServiceConfigurationCommand.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      {
+        return _responseCreator.CreateFailureResponse<bool>(HttpStatusCode.BadRequest);
+      }
+
+      List<Guid> changedServicesId = await _repository.EditAsync(servicesId);
```
Comment (MariyaSin) [inline src/AdminService.Business/Commands/EditServiceConfigurationCommand.cs]: servicesIds

File: src/AdminService.Business/Commands/EditServiceConfigurationCommand.cs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+      bool difference = changedServicesId.Count == servicesId.Count;
+
+      return new OperationResultResponse<bool>()
+      {
+        Status = difference ? OperationResultStatusType.FullSuccess : OperationResultStatusType.PartialSuccess,
+        Body = true,
+        Errors = difference ? null : new() { "Request contains incorrect Ids." }
```
Comment (MariyaSin) [inline src/AdminService.Business/Commands/EditServiceConfigurationCommand.cs]: Request contains incorrect services Ids.

File: src/AdminService.Business/Commands/Interfaces/IEditServiceConfigurationCommand.cs [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+using System.Threading.Tasks;
+using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.Kernel.Responses;
+
+namespace LT.DigitalOffice.AdminService.Business.Commands.Interfaces
+{
+  [AutoInject]
+  public interface IEditServiceConfigurationCommand
+  {
+    Task<OperationResultResponse<bool>> ExecuteAsync(List<Guid> servicesId);
```
Comment (MariyaSin) [inline src/AdminService.Business/Commands/Interfaces/IEditServiceConfigurationCommand.cs]: servicesIds

File: src/AdminService.Data/ServiceConfigurationRepository.cs [hunk 5]
```diff
@@ -30,5 +37,31 @@ public async Task<(List<DbServiceConfiguration> dbServicesConfigurations, int to
         await dbServicesConfigurations.Skip(filter.SkipCount).Take(filter.TakeCount).ToListAsync(),
         await dbServicesConfigurations.CountAsync());
     }
+
+    public async Task<List<Guid>> EditAsync(List<Guid> servicesId)
```
Comment (MariyaSin) [inline src/AdminService.Data/ServiceConfigurationRepository.cs]: servicesIds

File: src/AdminService.Data/Interfaces/IServiceConfigurationRepository.cs [hunk 6]
```diff
@@ -10,5 +11,6 @@ namespace LT.DigitalOffice.AdminService.Data.Interfaces
   public interface IServiceConfigurationRepository
   {
     Task<(List<DbServiceConfiguration> dbServicesConfigurations, int totalCount)> FindAsync(BaseFindFilter filter);
+    Task<List<Guid>> EditAsync(List<Guid> servicesIds);
```
Comment (VladSpitsin) [inline src/AdminService.Data/Interfaces/IServiceConfigurationRepository.cs:15]: add line

**Review by grinakate — APPROVED** [review]

**Review by ekaterinaryzhenkova — APPROVED** [review]

**Review by VladSpitsin — APPROVED** [review]

**Review by irlirla — APPROVED** [review]

**Pull request merged**